### DISCUSSION TITLE
argo-workflows/workflow-notifications/

### DISCUSSION THREAD

**Discussion by giscus** [argoproj/argo-workflows#10226] [General]
# argo-workflows/workflow-notifications/

https://argoproj.github.io/argo-workflows/workflow-notifications/

**Comment by ghost** [comment]
Could you please provide an example on 2nd option.

**Reply by OneCricketeer** [reply]
https://github.com/argoproj/argo-workflows/blob/master/examples/exit-handler-slack.yaml#L19-L36

**Comment by mbruner** [comment]
[Heptio Labs EventRouter](https://github.com/heptiolabs/eventrouter) is archived. Are there any other alternatives?

**Reply by OneCricketeer** [reply]
[Argo Events](https://argoproj.github.io/argo-events/)?

**Comment by enzo-dechaene** [comment]
Could you please provide an example on 3rd option.

**Comment by kevinpauli** [comment]
Lifecycle Hooks also seems like a good option now https://argoproj.github.io/argo-workflows/lifecyclehook/

**Comment by ampandey111** [comment]
Hello, Can you please any other alternative in place event router(as it is archived now) in step 3
